server/Makefile

```diff
@@ -156,7 +156,9 @@ PLUGIN_PACKAGES += mattermost-plugin-nps-v1.3.3
 PLUGIN_PACKAGES += mattermost-plugin-servicenow-v2.3.4
 PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.6.2
 
-EE_PACKAGES=$(shell $(GO) list $(BUILD_ENTERPRISE_DIR)/...)
+EE_PACKAGES=$(shell $(GO) list $(BUILD_ENTERPRISE_DIR)/elasticsearch)
+export MM_ELASTICSEARCHSETTINGS_USERNAME := admin
+export MM_ELASTICSEARCHSETTINGS_PASSWORD := admin
```

Comment:
This is obviously temporary. We will need a way to alternate between ES and OpenSearch in the test environment. AFAIK these credentials are required for OpenSearch. Maybe we can configure both ES and OpenSearch identically to accept the same login creds, or have separate configurations for each test case.